controllers/club/settings/invite_members/impl/emailNewMembers.js

```diff
@@ -43,13 +47,27 @@ module.exports = async (req, res) => {
                         const settings_message = {User: user.firstName + " " + user.lastName, Type: `Invited user via email to join club`, Date: date, Time: time}
                         club.settings_history.unshift(settings_message)
                         club.save()
+                        
+                        const oauth2Client = new OAuth2(
```

Comment:
Could we also break these up into separate functions?

Reply:
which part would you like to break into separate functions?